src/commands/codepush/patch.ts

```diff
@@ -0,0 +1,126 @@
+import { out } from "../../util/interaction";
+import { DefaultApp } from "../../util/profile";
+import { inspect } from "util";
+import { MobileCenterClient, models, clientRequest, clientCall } from "../../util/apis";
+const _ = require("lodash");
+const chalk = require("chalk");
+import * as semver from "semver";
+
+const debug = require("debug")("mobile-center-cli:commands:codepush:patch");
+
+@help("Update the metadata for an existing release")
+export default class PatchCommand extends AppCommand {
+  
+  @help("CodePush deployment name")
+  @required
+  @name("ExistingDeploymentName")
+  @position(0)
+  public deploymentName: string;
+
+  @help("CodePush release label")
+  @required
+  @name("ExistingReleaseLabel")
+  @position(1)
+  public releaseLabel: string;
+
+  @help("update whether the release should be considered mandatory or not")
+  @shortName("m")
+  @longName("mandatory")
+  @hasArg
+  public isMandatory: string;
+
+  @help("update the description associated with the release")
+  @shortName("x")
+  @longName("disabled")
+  @hasArg
+  public isDisabled: string;
+
+  @help("update the semver range that indicates which binary version(s) a release")
+  @shortName("t")
+  @longName("targetBinaryVersion")
+  @hasArg
+  public targetBinaryRange: string;
+
+  @help("update the description associated with the release")
+  @shortName("des")
+  @longName("description")
+  @hasArg
+  public description: string;
+
+  @help("allows you to increase the rollout percentage of the target release")
+  @shortName("r")
+  @longName("rollout")
+  @hasArg
+  public rollout: string;
+
+  constructor(args: CommandArgs) {
+    super(args);
+  }
+
+  async run(client: MobileCenterClient): Promise<CommandResult> {
+
+    const app = this.app;
+    let release: models.LiveUpdateRelease;
+
+    if (this.targetBinaryRange == null && this.isDisabled == null && this.isMandatory == null && this.description == null && this.rollout == null) {
+      return failure(ErrorCodes.Exception, "At least one property must be specified to patch a release.");
+    }
+
+    if (this.isMandatory != null) {
+      if (this.isMandatory != 'true' && this.isMandatory != 'false') {
+        return failure(ErrorCodes.Exception, `Mandatory value should be either ${chalk.bold(true)} or ${chalk.bold(false)}.`);  
+      }
+    }
+
+    if (this.isDisabled != null) {
+      if (this.isDisabled != 'true' && this.isDisabled != 'false') {
+        return failure(ErrorCodes.Exception, `Disabled value should be either ${chalk.bold(true)} or ${chalk.bold(false)}.`);  
+      }
+    }
+
+    if (this.rollout != null) {
+      if (parseInt(this.rollout) < 0 || parseInt(this.rollout) > 100) {
+        return failure(ErrorCodes.Exception, `Rollout value should be integer value between ${chalk.bold(0)} or ${chalk.bold(100)}.`);
+      }
+    }
+
+    const isValidVersion = (version: string): boolean => !!semver.valid(version) || /^\d+\.\d+$/.test(version);
```

Comment:
`const isValidVersion` check is out of date here, should be `const isValidVersion = (version: string): boolean => !!semver.valid(version) || /^\d+\.\d+$/.test(version) || /^\d+$/.test(version);`
Please take a look at https://github.com/Microsoft/code-push/blob/c5a294d503226536b3022b9e0c86303c2d14e29d/cli/script/command-executor.ts#L902